data/core/step3_rasa_x.md

```diff
@@ -0,0 +1,11 @@
+## Get started from website
+* get_started_step3
+    - action_greet_user
```

Comment:
Can you start step 3 from the website, or just step 2?

Also, where's the slot event?

Reply:
I'm not sure about this. @akelad Could you comment on this?